lib/prometheus/business_collector.rb

```diff
@@ -1,212 +0,0 @@
-# frozen_string_literal: true
-
-unless defined? Rails
-  require File.expand_path('../../config/environment', __dir__)
-end
-
-# Collects stats relevent for the business from the database
-class BusinessCollector < PrometheusExporter::Server::TypeCollector
```

Comment:
Can we remove this separately? These might be reworked (or not) to new exporter. If we remove them now they will be forgotten pretty quickly (or is that the intention)?

Reply:
Sure

Reply:
I want to rework them in a followup, for now I extracted it to a separate commit.

Reply:
Marked it [here](https://issues.redhat.com/browse/RHICOMPL-2582)

Reply:
Without the prometheus_exporter gem (that causes the memory leak) the tests would fail anyway and an accidental autoload (which has almost 0 chance) could crash the pods.